packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts

```diff
@@ -3,6 +3,7 @@
 
 const navigateToSettingsAndBack = () => {
     cy.getTestElement('@analytics/consent');
+    cy.wait(5000); // needed to prevent anunwanted redirect to /onboarding
```

Comment:
I am still not happy about this, investigating a proper fix.

Reply:
Sometimes, when I open Suite on this branch, I am redirected right to onboarding. This is a bug (I should see the welcome screen) and it is probably related to why this test was failing. Also investigating why it is happening and in what circumstances...

https://github.com/trezor/trezor-suite/assets/42465546/664d3b39-f09b-4511-a6db-ba73415b2717